src/angular-app/languageforge/lexicon/directive/dc-semanticdomain.js

```diff
@@ -12,9 +12,8 @@ angular.module('palaso.ui.dc.semanticdomain', [])
       model: '=',
       selectField: '&'
     },
-    controller: ['$scope', '$state', 'lexRightsService', function ($scope, $state, rights) {
+    controller: ['$scope', '$state', 'lexRightsService', function ($scope, $state, rightsService) {
```

Comment:
The `lexRightsService` uses the `sessionService`, adding additional logic, such as whether a project is archived, and whether a send and receive is in progress (thus disallowing certain functions that would otherwise be allowed).